### ISSUE TITLE
Expand/Collapse Images coded as CSS background images.

### ISSUE THREAD

**Issue by davidstenger** [uswds/uswds#1645]
**Issue:**
CSS images are used to convey expand and collapse functionality.  Users who navigate to the page with images disabled, an alternate style sheet applied, or Windows High Contrast Mode applied also do not see these expand/collapse CSS background images. 
See attachment  [CSSImage_ExtendedHeader.png](https://drive.google.com/open?id=0ByQeU_iOqP_iN0dscnhpb1ZOS2c)

Also, in responsive view, the Plus and minus icons are missing in disabled image view. ResponsiveView_CSSImage_ExtendedHeader.png

**Solution:**
Choose one of the following options to resolve this issue:
Option 1: Instead of using CSS to display the expand and collapse images, use the HTML <img> tag with an appropriate alt attribute defined (e.g., alt=”Expand” and “alt=”Collapse”) and include the image within the <a> tag for the link it is associated to.

Option 2:  Keep the expand/collapse images as CSS background images but then add hidden text using CSS absolute positioning. The hidden text would be either "Expand" or "Collapse" (depending on the state of the control) and include it within the <a> tag for the link. IN ADDITION, you would need to implement a High Contrast Mode Detection script so that the images are also visible in High Contrast Mode, with an alternate stylesheet, etc. script is located at the link

**Comment by maya** [comment]
This is a challenging one! We could do #1 using an svg in the HTML but it would be a little more work for developers in the HTML. Still, I think it's probably better than needing a high contrast mode detection script and using a separate CSS style for that.

**Comment by thisisdano** [comment]
Thanks for contributing to USWDS. Unfortunately, this issue is not on our current [product roadmap](https://designsystem.digital.gov/whats-new/product-roadmap/).
